daemon/src/rollover/maker.rs

```diff
@@ -249,7 +253,8 @@ impl Actor {
 
         emit_rejected(order_id, &self.executor).await;
 
-        self.tasks.add_fallible(
+        let mut tasks = Tasks::default();
+        tasks.add_fallible(
```

Comment:
Not an issue imo, but this lets the maker cancel an ongoing rollover by quickly rejecting it after having accepted.

Reply:
interesting! this might actually be a feature 😁